docs/usage.rst

```diff
@@ -0,0 +1,197 @@
+Naggum usage
+============
+
+Currently there are two dialects of Naggum: high level *Compiler* and low-level
+*Assembler*.
+
+Naggum Compiler
+---------------
+
+Command line syntax for Naggum Compiler is::
+
+    $ Naggum.Compiler [source1.naggum] [source2.naggum] [/r:assembly1] [/r:assembly2]
```

Comment:
(Maybe you're just using a different commandline convention; mine stems from Linux manpages.)

This line appears to suggest that you can pass at most two source files to the Compiler. I suggest rewriting it like this:
```
$ Naggum.Compiler source.naggum... [/r:assembly1]...
```
(assuming the flags are optional and at least one source filename is required.)